episodes/10-defensive.md

```diff
@@ -467,9 +467,9 @@ can you think of a function that will pass your tests but not his/hers or vice v
 
 ```python
 # a possible pre-condition:
-assert len(input_list) > 0, 'List length must be non-zero'
+assert len(input_array) > 0, 'List length must be non-zero'
```

Comment:
Thanks @fiveop for the suggestion and contribution!
Please note that there is one more `list` term to be replaced with `numpy array` on this line.

While reviewing the code, I'd like to suggest considering the data type of the variable `input_list`. If it happens to be a standard Python list instead of a NumPy array, it might be beneficial to consistently refer to it as a 'list' in our comments to maintain clarity. The decision between 'list' and 'array' should align with the actual type of the variable at hand. 

To enhance comprehension for both teachers and students, it might be helpful to include a brief comment or explanation at certain points in the code. As you've initiated the pull request, I wonder if it might be appropriate to insert comments or descriptions where you think they would be most effective. Please feel free to do so, and if there's any need for assistance from a maintainer's end, please don't hesitate to reach out. 🙂

Reply:
I changed the assertion message to use the term array instead of list.

I am not sure whether I understand correctly, where you suggest to add further comments. If it is in the solution of this exercise, I would suggest to not do that, because it would distract from what the exercise is about: thinking about pre- and post-conditions.